File: aiJ-client/assets/Script/plazz/event/RechargeRecordEvent.ts
import {AiJ} from "../../ws/AiJ";

/**
 * 交易记录
 */
export default class RechargeRecordEvent extends AiJ.AiJEvent {

    /**
     * 页码
     */
    page: number;
    /**
     * 每页显示条数
     */
    pageSize: number;
    /**
     * 类型
     */
    assetCode: string;

    constructor(page: number, pageSize: number, assetCode: string) {
        super();
        this.mainType = 4;
        this.subType = 3;
        this.page = page;
        this.pageSize = pageSize;
        this.assetCode = assetCode;
    }
}

File: aiJ-plaza/src/main/java/com/xiyoufang/aij/plaza/event/RechargeRecordEvent.java
package com.xiyoufang.aij.plaza.event;

import com.xiyoufang.aij.event.Event;

/**
 * Created by 席有芳 on 2019-03-01.
 *
 * @author 席有芳
 */
public class RechargeRecordEvent extends Event {
    /**
     * 页码
     */
    private int page;
    /**
     * 每页显示条数
     */
    private int pageSize;
    /**
     * 类型
     */
    private String assetCode;

    public int getPage() {
        return page;
    }

    public void setPage(int page) {
        this.page = page;
    }

    public int getPageSize() {
        return pageSize;
    }

    public void setPageSize(int pageSize) {
        this.pageSize = pageSize;
    }

    public String getAssetCode() {
        return assetCode;
    }

    public void setAssetCode(String assetCode) {
        this.assetCode = assetCode;
    }
}
